clients/cpp-console/include/AudioPlayerEntry.h

```diff
@@ -1,25 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#include "speechapi_cxx.h"
-using namespace Microsoft::CognitiveServices::Speech;
+#include "AudioPlayerStream.h"
 
 namespace AudioPlayer
 {
     enum class PlayerEntryType
     {
         BYTE_ARRAY,
-        PULL_AUDIO_OUTPUT_STREM
+        PULL_AUDIO_OUTPUT_STREAM
     };
 
     class AudioPlayerEntry
     {
     public:
         AudioPlayerEntry(unsigned char* pData, size_t pSize);
-        AudioPlayerEntry(std::shared_ptr<Audio::PullAudioOutputStream> pStream);
+        AudioPlayerEntry(std::shared_ptr<IAudioPlayerStream> pStream);
 
         PlayerEntryType m_entryType;
-        std::shared_ptr<Audio::PullAudioOutputStream> m_pullStream;
+        std::shared_ptr<IAudioPlayerStream> m_AudioPlayerStream;
```

Comment:
the format is that the 'A' should be an 'a'

Reply:
Done